src/docs/concepts-development/sitemap/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Sitemap support in Sharetribe
+slug: sitemap-in-sharetribe
+updated: 2023-10-12
+category: concepts-development
+ingress:
+  This guide describes how the Sharetribe tooling supports creating a
+  sitemap for your marketplace.
+published: true
+---
+
+Having a sitemap on your marketplace helps search engines to process
+your website correctly. For a traditional website, sitemaps are
+relatively easy to make, because all of your routes are built in to the
+application.
+
```

Comment:
Comment:
Maybe we could mention that we provide API endpoints (and link to them) for generating dynamic sitemaps already in the introduction section right now they're only mentioned under "Separate sitemaps for static and dynamic content"